project2/styles.css

```diff
@@ -0,0 +1,173 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(255, 255, 255);
+    color: rgb(0, 0, 0);
+    font-family: 'Playfair Display', serif;
+    font-size: 17px;
+}
+
+
+.menu li {
+    display: inline-block;
+}
+
+ul {
+    text-align: left;
+    list-style: square;
+    padding-left: 60px;
+    padding-right: 40px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+}
+
+.menu li+li {
+    margin-left: 110px;
+}
+
+li a {
+    background-color: rgb(255, 255, 255);
+    font-family: monospace;
+    color: orange;
+    font-size: 35px;
+    padding-left: 32px;
```

Comment:
You want to be careful adding padding like this to an `a` element because it means there's now 32 pixels of blank space to the left of your icon that is _also_ clickable.

If what you're trying to achieve is centering the icons, you may instead want to add this to your CSS:
```
.menu {
    text-align: center;
}
```